### PR TITLE
Information Display enhancements

### PR DESCRIPTION
**Pull request by paroxp**
## What

We may want to keep the current template of the board, which may prevent people from being lost or scroll too much.

The syntax has been somewhat cleaned up, some of the information either hidden or deleted.

Unfortunately, some javascript has been added. I am terribly sorry about that.

## How to review

Fire up the application and checkout my approach. Perhaps, ask people around for their opinion too.

Feel free, to give me a smack on the back of the head.

### CODE REVIEW

**Comment by Rory80Hz** [comment]
Cool! I'll check it out.

File: views/index.ejs [hunk 1]
```diff
@@ -1,52 +1,72 @@
...
-    <% if (reviewSlotsFull) { %>
-    <p class="text-warning lead">
-      <img src="/images/sad_panda.gif" alt="Sad Panda" class="img-circle">&nbsp;I Am a sad panda.  I will never find any slots for review.
-    </p>
-    <% } %>    
-    <% story.forEach(function(story){ %>
-      <%- include('story/summary', {story: story, status: "started", message: "In Progress"}); %>
-      <% }); %>
+        <div class="col-md-4">
+            <h2>Play</h2>
```
Comment (Rory80Hz) [inline views/index.ejs]: Lets Label these 'Doing' 'Reviewing' 'Approving'

File: views/index.ejs [hunk 2]
```diff
@@ -1,52 +1,72 @@
...
+            <a href="https://www.pivotaltracker.com/n/projects/<%= projectId %>" target="_blank">
+                Full Fat Pivotal Buzz
+            </a>
         </p>
     </footer>
-    </div><!-- /container -->
-  </body>
+</div><!-- /container -->
+
+<!-- I am terribly sorry for this... Really sorry... - Rafal -->
```
Comment (Rory80Hz) [inline views/index.ejs]: lol, that's fine. let's include it in source though, instead of via the CDN, add it to public/js/

**Review by Rory80Hz — CHANGES_REQUESTED** [review]
Looks good, just a couple of minor tweaks!

File: views/index.ejs (hunk 1 above)
Comment (paroxp) [reply views/index.ejs]: Certainly.

File: views/index.ejs (hunk 2 above)
Comment (paroxp) [reply views/index.ejs]: Sure thing :)

**Comment by paroxp** [comment]
Just FYI, I did the above requested changes. Sorry, if you're waiting on me :(

**Pull request closed without merging**